### DISCUSSION TITLE
Filter usability - operators list formatting

### DISCUSSION THREAD

**Discussion by jdjoshuadavison** [directus/directus#19720] [Feature Requests]
### The overarching problem
Users find the search and filters difficult to use.
This is one of a number of requests to help solve this problem.
There are 2 ideas in one here.

### List formatting
To help make the list of operators a little quicker and easier to scan, I suggest some formatting that groups the variations of operators (inverse operators and case sensitive) underneath the main operator. Done right, this will make it easier for the user to quickly identify and scan the unique operators whilst keeping the variations easy to find.

### Motivation

Users find the filters difficult to use.

### List formatting
There could be a number of design solutions, but for my example I've simply used an indent. A simple indent on the operator variations makes the list a lot quicker to scan. At a glance you can immediately identify 3 unique operators and that they have variations. Once you've found the operator you're looking for, you can look quickly and easily scan the list of variations to find the specific variation you're looking for:
![filter-ui-operator-formatting](https://github.com/directus/directus/assets/34007858/12e3e5fe-c208-4840-864c-34b3e39476a0)

### Must have
- List formatting

### Drawbacks

None.

### List formatting
There could be plenty of other ways of achieving this "grouping" effect. For example you could use a dash or some other icon instead of indenting, or you could use background colours and or borders.

Another alternative - **or additional** - solution could be to remove case sensitive options and add a case sensitive to the filter rule row. See: #19721 

### List formatting
None needed - visual change only.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
